packages/blade/src/components/Chip/_decisions/decisions.md

```diff
@@ -0,0 +1,127 @@
+| **isDisabled**          | `boolean`                                                            | `false`     | Disables or enables the Chip                                                                                                                       | ❌       |
+| **size**                | `'xsmall' \| 'small' \| 'medium' \| 'large'`                         | `'small'`   | Specifies the size of the rendered Chip                                                                                                            | ❌       |
+| **value (html native)** | `string`                                                             | `undefined` | The value of the input field in Chip, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#value) | ❌       |
+| **variant**             | `'positive' \| 'negative' \| 'notice' \| 'information' \| 'neutral'` | `undefined` | Sets the Chip's visual variant                                                                                                                     | ❌       |
+
+## `ChipGroup` API
+
+| Prop              | Type                                                                     | Default     | Description                                                                                                                    | Required |
+| ----------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| **children**      | `React.ReactNode`                                                        | `undefined` | Accepts multiple `Chip` components                                                                                             | ✅       |
+| **defaultValue**  | `string \|string[]`                                                      | `undefined` | Sets the initial value of the Chip group                                                                                       | ❌       |
+| **isDisabled**    | `boolean`                                                                | `false`     | Disables or enables ChipGroup, it will propagate down to all the Chips                                                         | ❌       |
+| **name**          | `string`                                                                 | `undefined` | The name of the chip group, [useful in form submissions](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#name) | ❌       |
+| **onChange**      | `({ name, value }: { name: string; value: string \| string[] }) => void` | `undefined` | Calls a function on any state change within the ChipGroup                                                                      | ❌       |
```

Comment:
@kamleshchandnani The type of `value` depends on the `selectionType` prop. If its `single` value will be of `string` type and if it's `multiple` it will be `string[]` simulating radio/checkbox-like behavior respectively. So users won't have to add a check like `Array.isArray(value)`, since it will be either one of the cases only.

Reply:
yes i understand that so the call we took in select was that we'll always make this array so if its single value the array will always have single value. because imagine if you make the dropdown multiple/single at runtime based on API response or if you change it manually in code you don't have to worry about the onchange function.so better DX and less error prone

Reply:
@kamleshchandnani I see, makes sense. I'll update it.